### PR TITLE
Ramsey UUID TypeDescriptor

### PR DESCRIPTION
**Pull request by mhujer**

### CODE REVIEW

File: src/Type/Doctrine/Descriptors/Ramsey/UuidBinaryOrderedTimeTypeDescriptor.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Type\Doctrine\Descriptors\Ramsey;
```
Comment (mhujer) [inline src/Type/Doctrine/Descriptors/Ramsey/UuidBinaryOrderedTimeTypeDescriptor.php]: What about putting third-party descriptors into respective subnamespaces?

File: src/Type/Doctrine/Descriptors/Ramsey/UuidBinaryOrderedTimeTypeDescriptor.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+use PHPStan\Type\Doctrine\Descriptors\DoctrineTypeDescriptor;
+use PHPStan\Type\Type;
+use PHPStan\Type\TypeCombinator;
+use Ramsey\Uuid\Doctrine\UuidBinaryOrderedTimeType;
+use Ramsey\Uuid\UuidInterface;
+
+class UuidBinaryOrderedTimeTypeDescriptor implements DoctrineTypeDescriptor
+{
+
+	public function getType(): string
```
Comment (mhujer) [inline src/Type/Doctrine/Descriptors/Ramsey/UuidBinaryOrderedTimeTypeDescriptor.php]: All classes are identical except for the `getType()` method

Comment (ondrejmirtes) [reply src/Type/Doctrine/Descriptors/Ramsey/UuidBinaryOrderedTimeTypeDescriptor.php]: Instead, inject the type into the constructor and register the same class as three different services :)

Also a test case would be nice, you can add one of the descriptors to EntityColumnRuleTest and add an entity field with UUID type there :)

**Comment by lookyman** [comment]
I was thinking about creating a separate package for this. But considering the popularity of Ramsey's extension, I guess we can have this in here?

**Comment by ondrejmirtes** [comment]
It doesn’t have any dependency nor downside, I want it here 😊

**Comment by ondrejmirtes** [comment]
Thank you!

**Pull request merged**